region/lowernorfair/west.json

```diff
@@ -454,12 +454,54 @@
               "id": 3,
               "strats": [
                 {
-                  "name": "Base",
+                  "name": "GT Crumble Jump to Item",
```

Comment:
make-able jump starting from inside the doorway

Reply:
Indeed.